SparkGeo1.ipynb

```
@@ -2,61 +2,82 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
    "id": "55a341cf",
    "metadata": {},
    "outputs": [],
```

Ruff (0.16.0)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)